devdocs/troubleshooting-guide.md

```diff
@@ -0,0 +1,198 @@
+# Developer Troubleshooting Guide
+
+1. [Troubleshooting project setup](#troubleshooting-project-setup)
+    * [Setup checklist](#setup-checklist)
+    * [Common errors and solutions](#common-errors-and-solutions)
+2. [Troubleshooting test failures](#troubleshooting-test-failures)
+    * [Optimizing IDE layout for testing](#optimizing-ide-layout-for-testing)
+    * [How automated browser tests work](#how-automated-browser-tests-work)
+    * [Getting help to resolve test failures](#getting-help-to-resolve-test-failures)
+    * [Other errors and solutions](#other-errors-and-solutions)
+        * [Errors specific to Linux](#errors-specific-to-linux)
+
+This document can help you to fix the problems encountered while contributing to TEAMMATES.Failing that, you can post in the [issue tracker](https://github.com/TEAMMATES/teammates/issues/new) to ask for help.Remember to supply as much relevant information as possible when asking for help. e.g. Which operating system are you using.
```

Comment:
The link could wrap the entire "post in the issue tracker" instead of just "issue tracker". Also, don't forget space after each sentence ends (there are two missing here).